src/store/store.ts

```diff
@@ -103,7 +103,7 @@ export class Store {
 				}
 			}
 		} catch (error) {
-			console.error(`Failed to open last styleguide or page: ${error}`);
+			throw new Error(`Failed to open last styleguide or page: ${error}`);
```

Comment:
This one should stay as a `console.error`. :)

Reply:
Understandable. I didn't want to leave any caught but unhandled errors where I had removed a console.warn or console.error. But I'll restore all of these.